test/e2e/e2e-helpers.js

```diff
@@ -86,6 +86,12 @@ function expectNavigateAwayFrom(client, urlSubstring) {
     .to.not.contain(urlSubstring).before(Timeouts.slow);
 }
 
+// Expects navigation lands at a path with the given `urlSubstring`.
+function expectNavigateAwayFromExact(client, urlSubstring) {
```

Comment:
:thinking: So this is beyond the scope of this PR, but I'm pondering if it'd be useful to have some sort of `expectLocationToBe()` helper for debugging, like we ran into the other day.

Reply:
Great idea, @jbalboni would you care to provide any insights into this as well?

Reply:
Yeah, I think we would benefit from some better helpers around the page location.